config/opensearch_dashboards.yml

```diff
@@ -285,4 +285,8 @@
 # opensearchDashboards.survey.url: "https://survey.opensearch.org"
 
 # Set the value of this setting to true to enable plugin augmentation on Dashboard
-# vis_augmenter.pluginAugmentationEnabled: true
\ No newline at end of file
+# vis_augmenter.pluginAugmentationEnabled: true
+
+# Set the value to true to enable permission control for saved objects
+# Permission control depends on OpenSearch Dashboards has authentication enabled, set it to false if no authentication is configured
```

Comment:
What's the configuration for OSD authentication flag?

Reply:
should be security plugin installed or not

Reply:
Yeah, there's no authentication flag, I've modified the comment to make it more clear.